CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+CHANGES IN VERSION 1.11.0 (from 1.10.0)
+===================================
+
+Appium 1.11.0 is a minor release
+
+#### General
+
+#### iOS
+
+#### Android
+* Add `chromedriverDisableBuildCheck` capability [#474](https://github.com/appium/appium-android-driver/pull/474)
```

Comment:
when noting new caps it might be nice to say a few words about what they are used _for_, not just their name?